src/main/java/be/seeseemelk/mockbukkit/MockUnsafeValues.java

```diff
@@ -64,14 +68,16 @@ public ItemStack modifyItemStack(ItemStack stack, String arguments)
 		throw new UnimplementedOperationException();
 	}
 
+	private static final Set<String> COMPATIBLE_API_VERSIONS = new HashSet<>(Arrays.asList("1.13", "1.14", "1.15", "1.16"));
```

Comment:
Can you please put the field to the top of the class?
Also, there is no need for this to be static. Not an actual issue, just not necessary.

Reply:
Sure, no problem.